accepted/2024/[TS]PMUI-Readme-rendering.md

```diff
@@ -0,0 +1,216 @@
+This will be a link to download the README and will only be filled if a readme is available to download.
+
+###### index.json
+
+```json
+{
+    "version": "3.0.0",
+    "resources": [
+        ...,
+        {
+            "@id": "https://apidev.nugettest.org/v3/registration5-gz-semver2/",
+            "@type": "RegistrationsBaseUrl/6.12.0"
+        },
+        ...
+    ]
+}
+```
+###### Example Response
+```json
+{
+    "@id": "https://apidev.nugettest.org/v3/registration5-gz-semver2/newtonsoft.json/index.json",
+    "items": [
+        {
+            ...,
+            "items": [ 
+                ...,
+                {
+                    "@id": "https://apidev.nugettest.org/v3/registration5-gz-semver2/newtonsoft.json/13.0.3.json",
+                    "@type": "Package",
+                    "catalogEntry": {
+                        "@id": "https://apidev.nugettest.org/v3/catalog0/data/2024.08.19.16.55.59/newtonsoft.json.13.0.3.json",
+                        "@type": "PackageDetails",
+                        "authors": "James Newton-King",
+                        "dependencyGroups": [ ... ],
+                        "description": "Json.NET is a popular high-performance JSON framework for .NET",
+                        "iconUrl": "https://apidev.nugettest.org/v3-flatcontainer/newtonsoft.json/13.0.3/icon",
+                        "id": "Newtonsoft.Json",
+                        "language": "",
+                        "licenseExpression": "MIT",
+                        "licenseUrl": "https://dev.nugettest.org/packages/Newtonsoft.Json/13.0.3/license",
+                        "readmeUrl": "https://dev.nugettest.org/packages/Newtonsoft.Json/13.0.3#show-readme-container",
+                        //New field
+                        "readmeFileUrl": "https://apidev.nugettest.org/v3/flatcontainer/newtonsoft.json/13.0.3/readme",
+                        "listed": true,
+                        "minClientVersion": "2.12",
+                        "packageContent": "https://apidev.nugettest.org/v3-flatcontainer/newtonsoft.json/13.0.3/newtonsoft.json.13.0.3.nupkg",
+                        "projectUrl": "https://www.newtonsoft.com/json",
+                        "published": "2023-04-25T14:48:53.817+00:00",
+                        "requireLicenseAcceptance": false,
+                        "summary": "",
+                        "tags": [
+                            "json"
+                        ],
+                        "title": "Json.NET",
+                        "version": "13.0.3"
+                    },
+                    "packageContent": "https://apidev.nugettest.org/v3-flatcontainer/newtonsoft.json/13.0.3/newtonsoft.json.13.0.3.nupkg",
+                    "registration": "https://apidev.nugettest.org/v3/registration5-gz-semver2/newtonsoft.json/index.json"
+                },
+                ...
+            ],
+            ...
+        }
+    ],
+    ...
+}
+```
+##### ReadmeUriTemplate Resource Type
```

Comment:
Did we consider instead introducing a new version of the `PackageBaseAddress` resource that has this well known URI? I don't feel strongly about this, just a thought

Reply:
We originally planned for a new version of `PackageBaseAddress` but instead opted for `ReadmeUriTemplateResource`. 

There were questions about version, if we create PackageBaseAddress/6.13.0, then in the future implement a new one, PackageBaseAddress/6.20.0 that added another file, what that mean that we could use 6.20.0 to download the readme as well? Would a private feed have to implement a readme endpoint even if they don't support readme? 

Additionally, by setting using a different resource type feeds can specify a URI schema that better suites their implementation.